app/src/main/java/org/fossasia/openevent/general/settings/SettingsFragment.kt

```diff
@@ -22,6 +22,7 @@ class SettingsFragment : PreferenceFragmentCompat(), PreferenceChangeListener {
     private var email: String? = null
     val EMAIL: String = "EMAIL"
     val FORM_LINK: String = "https://docs.google.com/forms/d/e/1FAIpQLSd7Y1T1xoXeYaAG_b6Tu1YYK-jZssoC5ltmQbkUX0kmDZaKYw/viewform"
+    val SITE_LINK: String ="http://support.eventyay.com"
```

Comment:
This site does not tell how to use the app.

Reply:
Is there any other site? I found this most similar for this purpose having section like event setup, managing orders, payment, ticketing etc.

Reply:
for now we dont have a site that tells how to use the app

Reply:
So should i change the title to support site or else. I also see the potential in this site for our required purpose in future

Reply:
the site is a general FAQ site

Reply:
Ok @simarsingh24  should I close this PR or change How to use the app to FAQ till we don't get a dedicated site for this purpose.